### PR TITLE
Commtting yaml file  for Admin module

### PR DESCRIPTION
**Pull request by Navneetkr-2**
contains yaml file for Admin module

### CODE REVIEW

File: Common/yaml/admin.yaml [hunk 1]
```diff
@@ -0,0 +1,319 @@
+openapi: 3.0.3
+info:
+  title: Smart Contact Manager - OpenAPI 3.0
+  description: The project was about creating smart contact manager. Which can be specifically used for bussiness model. This is a web application which can be used as a simple pluging to any existing project.
```
Comment (babugarnaik) [inline Common/yaml/admin.yaml]: Correct spelling of business

File: Common/yaml/admin.yaml [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+      url: http://swagger.io
+  
+paths:
+  /v1/contacts/admin/users/{userId}:
+    get:
+      tags:
+        - contacts
+      summary: Find contacts 
+      description: To get list of contatcs of specific user 
+      operationId: getListOFContact
```
Comment (babugarnaik) [inline Common/yaml/admin.yaml]: OF is not in camel case

File: Common/yaml/admin.yaml [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+             application/json:
+              schema:
+                type: string
+                example: 
+                  message: Internal Server Error
+                  
+                
+    delete:
+      tags:
+           - contacts
```
Comment (babugarnaik) [inline Common/yaml/admin.yaml]: tags should always start with uppercase character

File: Common/yaml/admin.yaml [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+      summary: Delete contacts 
+      description: To delete  specific user's contacts
+      operationId: dleteContactOfUeser
```
Comment (babugarnaik) [inline Common/yaml/admin.yaml]: correct spell of delete

File: Common/yaml/admin.yaml (hunk 1 above)
Comment (Navneetkr-2) [reply Common/yaml/admin.yaml]: fixed

File: Common/yaml/admin.yaml (hunk 2 above)
Comment (Navneetkr-2) [reply Common/yaml/admin.yaml]: fixed

File: Common/yaml/admin.yaml (hunk 3 above)
Comment (Navneetkr-2) [reply Common/yaml/admin.yaml]: fixed

File: Common/yaml/admin.yaml (hunk 4 above)
Comment (Navneetkr-2) [reply Common/yaml/admin.yaml]: Fixed, please review again

File: Common/yaml/admin.yaml [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+  - url: http://localhost:8080
+
+  
+paths:
+  /v1/contacts/admin/users/{userId}:
+    get:
+      tags:
+        - Contacts
+      summary: Find contacts 
+      description: To get list of contatcs of specific user
```
Comment (babugarnaik) [inline Common/yaml/admin.yaml]: Correct the spelling of contacts

File: Common/yaml/admin.yaml [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/contactList'
+                
+                example:
+                  - id: 101
+                    fullName: Naneet Kumar
+                    contacatNumber: 7546930695
```
Comment (babugarnaik) [inline Common/yaml/admin.yaml]: contact spelling is wrong. please correct it.

File: Common/yaml/admin.yaml [hunk 7]
```diff
@@ -0,0 +1,314 @@
...
+                example: 
+                  message: Internal Server Error
+                  
+                
+    delete:
+      tags:
+           - Contacts
+      summary: Delete contacts 
+      description: To delete  specific user's contacts
+      operationId: deleteContactOfUeser
```
Comment (babugarnaik) [inline Common/yaml/admin.yaml]: User spell is correct

File: Common/yaml/admin.yaml [hunk 8]
```diff
@@ -0,0 +1,314 @@
...
+                example: 
+                 message: Internal Server Error
+  
+  /v1/categories/admin:
+    post:
+      tags:
+           - Category
+      summary: Create category 
+      description: Create category for all users by Admin
+      operationId: CreateCategory
```
Comment (babugarnaik) [inline Common/yaml/admin.yaml]: At some places, operationId is camel case and some places it is not. Please make it consistent

File: Common/yaml/admin.yaml [hunk 9]
```diff
@@ -0,0 +1,314 @@
...
+          content:
+             application/json:
+              schema:
+                type: string
+                example: 
+                 message: Internal Server Error
+          
+components:
+  schemas:
+    contactList:
```
Comment (babugarnaik) [inline Common/yaml/admin.yaml]: some schemas start with uppercase and some schemas with lower case. Please keep it consistent

**Review by babugarnaik — CHANGES_REQUESTED** [review]

File: Common/yaml/admin.yaml (hunk 5 above)
Comment (Navneetkr-2) [reply Common/yaml/admin.yaml]: fixed

File: Common/yaml/admin.yaml (hunk 6 above)
Comment (Navneetkr-2) [reply Common/yaml/admin.yaml]: fixed

File: Common/yaml/admin.yaml (hunk 7 above)
Comment (Navneetkr-2) [reply Common/yaml/admin.yaml]: fixed

File: Common/yaml/admin.yaml (hunk 8 above)
Comment (Navneetkr-2) [reply Common/yaml/admin.yaml]: fixed

File: Common/yaml/admin.yaml (hunk 9 above)
Comment (Navneetkr-2) [reply Common/yaml/admin.yaml]: I have fixed it. Please review again.

**Review by babugarnaik — APPROVED** [review]

**Pull request merged**